tests/test_playbooks/repository_set_info.yml

```diff
@@ -0,0 +1,71 @@
+---
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  tasks:
+    - name: ensure test organization
+      include_tasks: tasks/organization.yml
+      vars:
+        organization_name: "Test Organization"
+        organization_state: "present"
+
+- hosts: localhost
+  collections:
+    - theforeman.foreman
+  gather_facts: false
+  vars_files:
+    - vars/server.yml
+  roles:
+    - role: content_rhel
+      vars:
+        foreman_organization: "Test Organization"
+        foreman_validate_certs: false
+        foreman_manifest_download: false
+        foreman_manifest_path: "{{ subscription_manifest_path }}"
```

Comment:
Where is this variable coming from? :)

Reply:
`foreman_manifest_path` is expected by the manifest role at https://github.com/theforeman/foreman-ansible-modules/blob/292dc967ee203660c4628cfcf3edf12fc0278590/roles/manifest/tasks/main.yml#L17

`subscription_manifest_path` comes from my `tests/test_playbooks/vars/server.yml` which is based on the template at https://github.com/theforeman/foreman-ansible-modules/blob/292dc967ee203660c4628cfcf3edf12fc0278590/tests/test_playbooks/vars/server.yml.example#L17

This seems to work fine and is consistent with what I've done for other tests in the past. Are you suggesting there is a reason to take a different approach?

Reply:
No, I was just wondering that this is actually defined in our example XD (which you have shown is, so I am silent)